jefferson/core/jffs2.py

```diff
@@ -0,0 +1,468 @@
+import binascii
```

Comment:
"core" is like "data": tells nothing, mix bag for lots of things. As it's the only module in core right now, it could be simply named `jffs2.py` without a package.

Reply:
I missed these comments. You're totally right, code adapted.